x-pack/platform/plugins/shared/fleet/server/services/output.ts

```diff
@@ -1188,7 +1208,7 @@ class OutputService {
       savedObjectType: OUTPUT_SAVED_OBJECT_TYPE,
     });
 
-    const outputSO = await this.encryptedSoClient.update<Nullable<OutputSOAttributes>>(
+    const outputSO = await this.soClient.update<Nullable<OutputSOAttributes>>(
```

Comment:
I get a bit confused about the internal `this.soClient` and the `soClient` being passes as a parameter. Why are both needed?

Reply:
Thanks @jillguyonnet! ~~Could you open an issue to address this tech debt so we don't lose track of it?~~